tests/integration/targets/redis_info/tasks/main.yml

```
@@ -0,0 +1,41 @@
+# Copyright: (c) 2020, Pavlo Bashynskyi (@levonet)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: redis_info - connect to master with default host/port
+  community.general.redis_info:
+    login_password: "{{ redis_password }}"
+  register: result
+
+- assert:
+    that:
+    - result is not changed
+    - result.info is defined
+    - result.info.tcp_port == master_port
+    - result.info.role == 'master'
+
+- name: redis_info - connect to master with all parameters
```

```suggestion
- name: redis_info - connect to master with all parameters
```
the module supports the check mode. Even if it's fiction and the module doesn't change anything, it will be better to add a check with `check_mode: yes`as well